sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/SubexpressionEliminationSuite.scala

```diff
@@ -47,46 +47,43 @@ class SubexpressionEliminationSuite extends SparkFunSuite with ExpressionEvalHel
 
```

Comment:
The changes in this file is to adapt the new API in `EquivalentExpressions`. e.g.
`getAllEquivalentExprs()` -> `getAllExprStates`
`getEquivalentExprs(oneA).size == 1` -> `getExprState(oneA).get.useCount == 1`
`getEquivalentExprs(oneA).exists(_ eq oneA)` -> `getExprState(oneA).exists(_.expr eq oneA)`